### PR TITLE
Disable fuse optimization option

### LINKED ISSUE #18876: yolov3报错
image

### PR DESCRIPTION
**Pull request by chengduoZH**
enable_sequential_execution + fuse_all_optimizer_ops results in a cycle structure in the computational graph.
This bug will take a long time to fix, so turn off this option first, and open it after fixing the bug.

fix https://github.com/PaddlePaddle/Paddle/issues/18876

### CODE REVIEW

File: paddle/fluid/framework/details/build_strategy.h [hunk 1]
```diff
@@ -89,7 +89,7 @@ struct BuildStrategy {
   bool fuse_elewise_add_act_ops_{false};
   // Fuse_all_optimizer_ops and fuse_all_reduce_ops require that gradients
   // should not be sparse types
-  bool fuse_all_optimizer_ops_{true};
+  bool fuse_all_optimizer_ops_{false};
```
Comment (luotao1) [inline paddle/fluid/framework/details/build_strategy.h:92]: 默认改成true？因为上面在async mode下会强制设为false？

File: paddle/fluid/framework/details/build_strategy.cc [hunk 2]
```diff
@@ -96,6 +96,12 @@ class ParallelExecutorPassBuilder : public ir::PassBuilder {
           << "fuse_all_optimizer_ops only work in Reducer mode.";
       strategy_.fuse_all_reduce_ops_ = false;
     }
+    if (strategy_.async_mode_) {
+      VLOG_IF(3, strategy_.fuse_all_optimizer_ops_)
+          << "Currently, fuse_all_optimizer_ops doesn't works under "
```
Comment (luotao1) [inline paddle/fluid/framework/details/build_strategy.cc]: doesn't works-》doesn't work

File: paddle/fluid/framework/details/build_strategy.h (hunk 1 above)
Comment (chengduoZH) [reply paddle/fluid/framework/details/build_strategy.h:92]: enable_sequential_execution + fuse_all_optimizer_ops会有环
这个bug修起来时间会比较长  先关掉这个选项，等修复了bug 再打开这个选项

File: paddle/fluid/framework/details/build_strategy.cc (hunk 2 above)
Comment (chengduoZH) [reply paddle/fluid/framework/details/build_strategy.cc]: Done

**Review by luotao1 — APPROVED** [review]
LGTM

**Pull request merged**